packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -0,0 +1,37 @@
+import {ResourceSnapshotsModel} from '@coveord/platform-client';
+import ResourceSnapshots from '@coveord/platform-client/dist/definitions/resources/ResourceSnapshots/ResourceSnapshots';
+
+export class Snapshot {
+  constructor(
+    private model: ResourceSnapshotsModel,
+    private snapshotClient: ResourceSnapshots
+  ) {}
+
+  async push(targetOrganizationId: string, developerNotes: string) {
```

Comment:
Personal pet peeve, but I prefer to have the `public` keyword explicitly set.

Reply:
I'll do that correction in #240